### PR TITLE
feat(pycontw-2020/assets/keynotes): add wenyu-su

### PR DESCRIPTION
**Pull request by FrankYang0529**
(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Other (add static files)**

## Description
**Describe what the change is**
Add WenYu-Su keynote static files to pycontw-2020/assets/keynotes.

### CODE REVIEW

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            "title": "",
+            "description": ""
+        },
+        "zh": {
+            "title": "",
+            "description": ""
+        }
+    },
+    "social": {
+        "link": "http://pirsquared.org/",
```
Comment (Lee-W) [inline src/static/pycontw-2020/assets/keynotes/wenyu-su.json:24]: These links should be updated as well

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "twitter": "https://twitter.com/ivanov",
+        "github": "https://github.com/ivanov"
+    },
+    "slido": "https://app.sli.do/event/adohlrml"
```
Comment (Lee-W) [inline src/static/pycontw-2020/assets/keynotes/wenyu-su.json:28]: This column should remain empty for now.

Comment (FrankYang0529) [reply src/static/pycontw-2020/assets/keynotes/wenyu-su.json]: Removed it. Thank you.

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "en": {
+            "title": "",
+            "description": ""
+        },
+        "zh": {
+            "title": "",
+            "description": ""
+        }
+    },
+    "social": {
```
Comment (Lee-W) [inline src/static/pycontw-2020/assets/keynotes/wenyu-su.json:23]: Could you ask whether Prof. su is willing to add his contact information?

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            "name": "Wen Yu Su",
+            "bio": ""
+        },
+        "zh": {
+            "name": "蘇文鈺",
+            "bio": "蘇文鈺現為成功大學資訊工程系教授，國立交通大學士，紐約大學坦登學院碩博士，史丹佛大學博士後研究，工研院電通所工程師，中華大學資訊工程系副教授。2011 年之前致力於神經網路訊號處理，生理訊號分析，視音訊處理器設計，多核心處理器設計，內嵌記憶體管理與電子系統層工具 OpenESL，2012 年之後轉為致力於音樂音訊處理，機器學習與虛擬管弦樂團。多年來，以提高學生實務實作能力為要旨。\n2013 年，有鑒於資訊科技與程式設計日益重要，學生的實作探究能力降低，成立 Program The World Association (PTWA)（社團法人中華民國愛自造者學習協會），2014 年從嘉義東石偏鄉開始，帶領團隊教導偏鄉弱勢中小學學生以程式設計。截至 2020 年，摩頂放踵，足跡遍及 12 個縣市，輔導的學校與非營利組織超過 70 個，受惠的國教學生與教師超過 2500 人。多年來，課程漸趨多元，教學計劃包含 Train The Trainer，看見家鄉，程式與自造巡迴課程，自走車競賽，Coding-X 偏鄉高中職生程式設計，樂塾，特教科技課程。每年舉辦活動超過 70 場。並積極參與成功大學不分系，弱勢招生，上機考特殊選才，線上課程與考試系統等等促進高教進步的工作．多年來，PTWA 以募捐所得進行免費公益教學，雖感艱辛，但是見到弱勢家庭孩子得以成長，頗為欣慰。\n2018 年，有鑒於人工智慧的日益重要，但是多數課程缺乏實務操作經驗，吸引學生投入之不足，仿照國外名校，投入開發以遊戲打怪過關的方式的課程與線上平台 - ML Game，2020 年，平台完成，並在成功大學資訊系順利開課，目前已有多所學校接洽推廣，並預計在 2020/2021 與台灣微軟合作推廣課程，並舉辦全國性機器學習比賽。\n除了一般學術著作外，另出版，「做孩子的重要他人」與「老師是孩子遲來的父母」，紀錄多年在偏鄉與弱勢家庭的孩子的輔導工作。"
+        }
+    },
+    "session": {
+        "en": {
```
Comment (Lee-W) [inline src/static/pycontw-2020/assets/keynotes/wenyu-su.json:14]: We'll need to follow the content of his talk. But we can wait until next merge request.

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+    "photo": "pycontw-2020/assets/keynotes/wenyu-su.png",
+    "speaker": {
+        "en": {
+            "name": "Wen Yu Su",
+            "bio": ""
```
Comment (Lee-W) [inline src/static/pycontw-2020/assets/keynotes/wenyu-su.json:6]: Gina just finished her final revision. Please update it when you have time.

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json (hunk 1 above)
Comment (FrankYang0529) [reply src/static/pycontw-2020/assets/keynotes/wenyu-su.json]: Removed it. Thank you.

File: src/static/pycontw-2020/assets/keynotes/wenyu-su.json (hunk 5 above)
Comment (FrankYang0529) [reply src/static/pycontw-2020/assets/keynotes/wenyu-su.json]: Added it. Thank you.

**Pull request merged**